src-tauri/src/user.rs

```diff
@@ -0,0 +1,47 @@
+use anyhow::Result;
+use async_trait::async_trait;
+use serde::{Deserialize, Serialize};
+use user_service::UserService;
+
+use crate::service::Service;
+
+pub mod user_service;
+
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
+pub struct User {
+    id: u32,
+    username: String,
+}
+
+pub struct UserManager {
+    user_service: Box<UserService>,
+}
```

Comment:
What's the use of the Manager layer? Seems useless, we can inject the Service directly into the app, something like that.

Reply:
When implementing the offline/online mode, I want each manager to be reponsible of what to call.